source/js/buyers-guide/editorial-content-index.js

```diff
@@ -6,7 +6,7 @@
  * and not included in `bg-main.compiled.js`
  */
 
-import "htmx.org";
+import htmx from "htmx.org";
```

Comment:
I don't know how we're using htmx, but the last time I did this upgrade I had to add:

```
window.htmx = require('htmx.org')
```

Reply:
Hey @jhonatan-lopes , thanks for sharing your insights. Unfortunately this didn't fix the issue. The issues I found after fixing `ReferenceError` (by `import htmx from "htmx.org";`) was the inconsistent behaviours between browsers and page loads. I dived in further and found out it was caused by the script being loaded asynchronously. I have updated the code accordingly to address this issue.

Reply:
Good to know! Thanks!